docs: add ToolJet to showcase

withshubh opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.

You can learn more about contributing to Docusaurus here: https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md

Happy contributing!

-->

## Motivation

The documentation of ToolJet uses Docusaurus. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

Josh-Cena reviewed (changes requested):
Thanks

- Please don't choose "favorite" yourself—that's for the maintainers to decide. I personally don't think this qualifies as favorite despite being very polished and complete.
- The URL should be pointing to `https://docs.tooljet.com/`, not a non-docusaurus page
- Removing the color mode toggle's icons is a bad idea. You'd rather use SVG icons or something else if you don't like the emojis.
- The admonitions don't work well in dark mode:
![image](https://user-images.githubusercontent.com/55398995/146845355-41ccee3c-dbe3-4fba-a61d-4b73258e854c.png)

withshubh commented:
@Josh-Cena 👋  Thanks for your review.
- Removed the favorite tag
- added the source to the docs page

We will work on the other two suggestions too.

Josh-Cena commented:
Thanks for choosing Docusaurus! Nice site👍
